tools/acorn-optimizer.js

```diff
@@ -1299,7 +1299,7 @@ if (extraInfoStart > 0) {
 // Collect all JS code comments to this array so that we can retain them in the outputted code
 // if --closureFriendly was requested.
 var sourceComments = [];
-var ast = acorn.parse(input, { ecmaVersion: 6, preserveParens: closureFriendly, onComment: closureFriendly ? sourceComments : undefined });
+var ast = acorn.parse(input, { ecmaVersion: 9, preserveParens: closureFriendly, onComment: closureFriendly ? sourceComments : undefined });
```

Comment:
I'd probably use modern version notation instead; pretty much nobody says "ES9" but rather "ES2018".

```suggestion
var ast = acorn.parse(input, { ecmaVersion: 2018, preserveParens: closureFriendly, onComment: closureFriendly ? sourceComments : undefined });
```

Reply:
Or might even switch to `2019` right away.

```suggestion
var ast = acorn.parse(input, { ecmaVersion: 2019, preserveParens: closureFriendly, onComment: closureFriendly ? sourceComments : undefined });
```